### PR TITLE
Implement census data migration

### PR DESCRIPTION
**Pull request by drewsamnick**
This is the code to transfer the data in the pegasus FORMS table to the new census_submissions table. Just in case we discover any issues in the migration I am creating a new table to track which rows in census_submissions came from which forms row. I don't plan to run this in production until I've changed the census forms on both /yourschool and hourofcode to write directly to census_submissions.

### CODE REVIEW

**Comment by drewsamnick** [comment]
[Backfill script to migrate data from FORMS to census schema](https://trello.com/c/cYND872r/47-backfill-script-to-migrate-data-from-forms-to-census-schema)

File: bin/oneoff/move_census_data.rb [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+      full_address: data['school_full_address_s'] || processed_data['location_street_address_s'],
+      validation_type: SchoolInfo::VALIDATION_NONE
+    }
+
+    # The form allowed invalid (non-numeric) zip codes but the model requires an integer greater than 0
+    # Rather than fail to migrate the full record we just remove any bad zip codes
+    attrs[:zip] = nil if attrs[:zip].to_i <= 0
+  end
+
+  puts "Creatng school_info with attrs: #{attrs}" if @verbose
```
Comment (aoby) [inline bin/oneoff/move_census_data.rb]: typo: `Creatng`

File: bin/oneoff/move_census_data.rb [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+  opts.on('-v', '--verbose', 'Print detailed information.') do
+    @verbose = true
+  end
+end.parse!
+
+unless @trial_run
+  puts "WARNING: This is not a trial run. This run will update the DB."
+  puts "Press ENTER to continue"
+  gets
+end
```
Comment (aoby) [inline bin/oneoff/move_census_data.rb:438]: 👍

File: dashboard/test/models/census/census_submission_form_map_test.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class Census::CensusSubmissionFormMapTest < ActiveSupport::TestCase
+  test "empty census submission form map" do
```
Comment (aoby) [inline dashboard/test/models/census/census_submission_form_map_test.rb]: nit: add expected result to the test case name, e.g. `empty census submission form map is invalid`

File: bin/oneoff/move_census_data.rb (hunk 1 above)
Comment (drewsamnick) [reply bin/oneoff/move_census_data.rb]: Fixed

File: dashboard/test/models/census/census_submission_form_map_test.rb (hunk 3 above)
Comment (drewsamnick) [reply dashboard/test/models/census/census_submission_form_map_test.rb]: Updated

**Pull request merged**